### PR TITLE
ALP: adding users with combustion (bsc#1218568)

### PR DESCRIPTION
**Pull request by tbazant**
### Description

This update describe how to add users with combustio

### Are there any relevant issues/feature requests?

* bsc#1218568

### CODE REVIEW

File: references/combustion-configuration.xml [hunk 1]
```diff
@@ -164,6 +164,24 @@ umount /home /mnt
...
+    <section xml:id="combustion-adding-users">
+      <title>Creating new users</title>
+      <para>
+        To add a new user account, you first need to create a hash string that represents the
+        user's password, for example:
+      </para>
+<screen>&prompt.user;<command>openssl passwd -6</command>
+Password: 
+Verifying - Password: 
+$6$SgEWQCT1mC06n5i8$edBXdfQ./eEp4sVHh37sCxO1yqRuEggAUPSGU6uA5nqtD1NHPucubjtCMgnrrHJTW/KCO6Db/cEFsldweoNBr1</screen>
```
Comment (lvicoun) [inline references/combustion-configuration.xml:177]: Please remove the password hash. We cannot guarantee that it won't be use by users and thus create a security issue. IMHO the command itself should be sufficient here, as the output is selfexplanatory.

**Review by lvicoun — CHANGES_REQUESTED** [review]
Hi Tomas, just a small nitpick.

File: references/ignition-configuration.xml [hunk 2]
```diff
@@ -468,9 +467,9 @@ storage:
     "users": [
       {
         "name": "root",
-        "passwordHash": "$6$PfKm6Fv5WbqOvZ0C$g4kByYM.D2B5GCsgluuqDNL87oeXiHqctr6INNNmF75WPGgkLn9O9uVx4iEe3UdbbhaHbTJ1vpZymKWuDIrWI1",
+        "passwordHash": "<replaceable>PASSWORD-HASH</replaceable>",
```
Comment (dariavladykina) [inline references/ignition-configuration.xml:470]:
```suggestion
        "passwordHash": "<replaceable>PASSWORD_HASH</replaceable>",
```

**Review by dariavladykina — APPROVED** [review]
LGTM, I only corrected one more instance of underscore in replaceables.

**Pull request merged**